src/js/S3Client.js

```diff
@@ -80,4 +85,19 @@ export default class S3Client {
         return this.client.putObject(params).promise();
     }
 
+    uploadObject(bucketName, prefixWithSlash, files) {
+        return new Promise((resolve, reject) =>
+            async.each(files, (file, callback) => {
```

Comment:
I would just use promise.all instead of the async package. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all it works the same as far as I can tell.

Reply:
It is cleaner, thank you :)